bin/bundle

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
```

Comment:
I've taken care of this on my refac/PR_9 branch, so the change will be evident when I merge (or rebase) back into master (because I created a branch off of refac/PR_9 to work on the minimax kata).